emails/sns.py

```diff
@@ -0,0 +1,95 @@
+import base64
+import logging
+import pem
+import six
+from urllib.request import urlopen
```

Comment:
Should try to refactor to use `requests` library like we do elsewhere in the app code?

Reply:
This would be nice and seem do-able. According to the [third comment](https://stackoverflow.com/questions/36516183/what-should-i-use-to-open-a-url-instead-of-urlopen-in-urllib3) on the question you can simply
```
response = requests.get(cert_url)
pemfile = response.raw.read()
```
`requests.get(cert_url)` returns a  `requests.models.Response` and doing `response.raw` returns a `urllib3.response.HTTPResponse` which you can `read()`.

Reply:
`urlopen` returns a `http.client.HttrpResponse` and its `read()` [returns the response body](https://docs.python.org/3/library/http.client.html#http.client.HTTPResponse.read) and `requests.get` returns a `requests.models.Response` and its `content` [does the same thing.](https://2.python-requests.org/en/master/user/quickstart/#binary-response-content)

Reply:
We may be able to drop this entirely if we change to basic auth.

Reply:
I filed https://github.com/mozilla/fx-private-relay/issues/363 for this.